### ISSUE TITLE
[CM-11] [zeus] [31.08.2015] Doesn’t recognise headphones on boot

### ISSUE THREAD

**Issue by diedricm** [LegacyXperia/local_manifests#1002]
Reproduce by connecting headphones before booting. The audio will be played through the speakers and not the headphones.
After disconnecting and connecting the headphones from the device this bug is fixed until the next boot.

I'm sorry for not providing logs for this, but I couldn’t find a meaningful part in the bloat of debug messages on boot.

**Comment by realcatgirly** [comment]
after the legacy xperia logo you can plug the headphones in and they will work

**Comment by diedricm** [comment]
Yes this works too, but when you turn on the device with headphones plugged in, they wont be picked up.
Its definitely not a very bad bug, but I thought I should mention it nonetheless.

**Comment by realcatgirly** [comment]
Or you reboot the device because complete freeze and dont replug the headphones in.

**Comment by realcatgirly** [comment]
Also present in Cyanogenmod 12.1 on urushi

**Comment by aichitho** [comment]
Same problem here on CM11 iyokan.
For me replugging and sometimes even restarting doesn't work!
Seems as the phone needs to be turned on for a certain time to recognize earphones (original Sony Ericsson ones)
